openlibrary/macros/DonateModal.html

```diff
@@ -0,0 +1,95 @@
+$def with(book, size="M", ids={})
+
+$ title = book.title
+
+<style type="text/css">
+.donateModal {
+  padding: 0px 10px;
+}
+div.coverFloatHead h2.override {
+  font-size: 1.2em;
+  font-weight: bold;
+}
+.donateModal ul li, .donateModal ol li {
+  font-size: .875em;
+}
+.donateModal .donate-info {
+  display: flex;
+}
+.donate-info img {
+  width: 75px !important;
+  margin-left: 20px;
+}
+ol.instructions li {
+  list-style: initial;
+  margin: 10px 30px;
+}
+.donateModal .options {
+  display: flex;
+  padding: 0px 60px;
+}
+.donateModal .options pre.address{
+  flex: 1;
+}
+.benefits li {
+  margin: 15px 35px;
+}
+.benefits li:before {
+  margin-top: 8px;
+  content: "";
+  width: 20px;
+  height: 20px;
+  position: absolute;
+  background-image: url(https://archive.org/images/check.png);
+  background-size: cover;
+  background-position: 50%;
+  margin-left: -25px;
+  transform: translateY(-50%);
+</style>
+
+<div class="hidden">
+  <div class="coverFloat" id="seeDonate">
+    <div class="coverFloatHead">
+      <h2 class="override">
+        Donate this book to the <a href="https://archive.org">Internet Archive</a> library.
+      </h2>
+      <a class="dialog--close">&times;<span class="shift">$_("Close")</span></a>
+    </div>
+    <hr>
+    <div class="donateModal">
+      <p class="cta">Hooray! You've discovered a title that's missing from our <a href="https://help.archive.org/hc/en-us/articles/360016554912-Borrowing-From-The-Lending-Library">library</a>. Can you help donate a copy?</p>
+
+      <ol class="instructions">
+        <li>If you own this book, you can <a href="http://www.fedex.com/il/shippingguide/packaging.html">mail</a> it to our address below.</li>
+        <li>You can also purchase this book from a vendor and ship it to our address:</li>
+      </ol>
+      <br>
+      <div class="options">
+      <pre class="address">
```

Comment:
Yeah, class names like this are too generic and very likely to conflict. So something like `donateModal__address` would be better.